index.html

```diff
@@ -0,0 +1,192 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="menu mobile-only">
+        <img id="hide-menu" onclick="toggle()" src="images/menu.png" alt="menu">
+        
+        <div class="d-none" id="toggle-menu">
+          <a onclick="toggle()">
+            <img src="images/close.svg" alt="close-btn">
+          </a>
+          <ul>
+            <li>
+              <a href="about.html">About</a>
+              <img src="images/arrow-left.png" alt="about-page">
+            </li>
+            <li>
+              <a href="#">Porgram</a>
+              <img src="images/arrow-left.png" alt="program-page">
+            </li>
+            <li>
+              <a href="#">Join</a>
+              <img src="images/arrow-left.png" alt="join-page">
+            </li>
+            <li>
+              <a href="#">Sponsor</a>
+              <img src="images/arrow-left.png" alt="sponsor-page">
+            </li>
+            <li>
+              <a href="#">News</a>
+              <img src="images/arrow-left.png" alt="news-page">
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="desktop-only">
+        <ul class="top-nav">
+          <li>
+            <img src="images/facebook.png" alt="facebook">
+          </li>
+          <li>
+            <img src="images/twitter.png" alt="twitter">
+          </li>
+          <li>
+            <a href="#">English</a>
+          </li>
+          <li>
+            <a href="#">My Page</a>
+          </li>
+          <li>
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+      </div>
+        
+      <div class="nav-bar desktop-only">
+        <div class="bot-nav">
+          <img src="images/logo.png" alt="conference-logo">
+  
+          <ul class="main-nav">
+            <li>
+              <a href="about.html">About</a>
+            </li>
+            <li>
+              <a href="#">Program</a>
+            </li>
+            <li>
+              <a href="#">Join</a>
+            </li>
+            <li>
+              <a href="#">Sponsor</a>
+            </li>
+            <li>
+              <a href="#">News</a>
+            </li>   
+          </ul>
+
+          <button type="button" class="nav-btn">
+            CC Campaign
+          </button>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home">
+      <header id="titles-home">
+        <img src="images/title.png" alt="Creative-Commons-Global-Summit-2015">
+        <!-- <h2>"Hello! Sharing world"</h2>
+        <h1>
+          CREATIVE COMMONS
+          <br/>
+          GLOBAL SUMMIT
+          <br/>
+          2015
+        </h1> -->
```

Comment:
- [ ] It would be better if you delete the commented code. Please fix this minor issue. 
🛠️ 💯